messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -89,5 +92,8 @@ class ModuleMessagingInApp internal constructor(
             application = diGraph.context.applicationContext as Application,
             organizationId = organizationId
         )
+
+        // if identifier is already present, set the userToken again
```

Comment:
Thanks for adding a comment here. I think a comment attached to this code is needed to document why we did this change. 

I would appreciate adding more to this comment describing *why* we set the user token again. A short description of this bug would be sufficient.